### PR TITLE
refactor: adjust folder structure for compiler

### PR DESCRIPTION
**Pull request by julianiff**
-> Splits Compiler in team specific folders and adjusts Codeowners

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -141,6 +141,9 @@ packages/app-bridge/src/window.d.ts @Frontify/content-and-blocks @Frontify/theme
 packages/cli/src/commands/ @Frontify/content-and-blocks @Frontify/themes-and-navigation @Frontify/apps-and-integrations
 packages/cli/src/errors/ @Frontify/content-and-blocks @Frontify/themes-and-navigation @Frontify/apps-and-integrations
 packages/cli/src/utils/ @Frontify/content-and-blocks @Frontify/themes-and-navigation @Frontify/apps-and-integrations
+packages/cli/src/utils/platformAppCompiler.ts @Frontify/apps-and-integrations
```
Comment (ragi96) [inline CODEOWNERS:144]:
```suggestion
packages/cli/src/utils/compilePlatformApp.ts @Frontify/apps-and-integrations
```

So its the same the others?

**Review by ragi96 — APPROVED** [review]
For blocks it looks good, thanks for taking care :)

**Review by SamuelAlev — APPROVED** [review]
Didn't check the CLI, just the owned file is ok ✅

**Review by olivereisenhut — APPROVED** [review]

**Review by Kenny806 — APPROVED** [review]
LGTM

**Pull request merged**